src/NumericalAlgorithms/Interpolation/InterpolatorRegisterElement.hpp

```diff
@@ -56,6 +56,37 @@ struct RegisterElement {
   }
 };
 
+/// \ingroup ActionsGroup
+/// \brief Invoked on the `Interpolator` ParallelComponent to deregister an
+/// element with the `Interpolator`.
+///
+/// This is called by `RegisterElementWithInterpolator` below.
+///
+/// Uses: nothing
+///
+/// DataBox changes:
+/// - Adds: nothing
+/// - Removes: nothing
+/// - Modifies:
+///   - `Tags::NumberOfElements`
+///
+/// For requirements on Metavariables, see `InterpolationTarget`.
+struct DeregisterElement {
+  template <
+    typename ParallelComponent, typename DbTags, typename Metavariables,
+    typename ArrayIndex,
+    Requires<tmpl::list_contains_v<DbTags, Tags::NumberOfElements>> = nullptr>
```

Comment:
[optional] I _think_ we've been preferring the `db::tag_is_retrievable_v` over the `list_contains_v` but :shrug:

Reply:
`db::tag_is_retrievable_v` is preferred over `tmpl::list_contains_v` if there is the possibility that the tag being searched for was inserted as a compute or reference tag in which case the former will be true, but not the latter